parameter/command/include/command/Parser.h

```diff
@@ -0,0 +1,228 @@
+
+    CommandReturn restoreConfiguration(const IRemoteCommand& remoteCommand, std::string& result);
+
+    CommandReturn setElementSequence(const IRemoteCommand& remoteCommand, std::string& result);
+
+    CommandReturn getElementSequence(const IRemoteCommand& remoteCommand, std::string& result);
+
+    CommandReturn setRule(const IRemoteCommand& remoteCommand, std::string& result);
+
+    CommandReturn clearRule(const IRemoteCommand& remoteCommand, std::string& result);
+
+    CommandReturn getRule(const IRemoteCommand& remoteCommand, std::string& result);
+
+    CommandReturn listElements(const IRemoteCommand& remoteCommand, std::string& result);
+
+    CommandReturn listParameters(const IRemoteCommand& remoteCommand, std::string& result);
+
+    CommandReturn dumpElement(const IRemoteCommand& remoteCommand, std::string& result);
+
+    CommandReturn getElementSize(const IRemoteCommand& remoteCommand, std::string& result);
+
+    CommandReturn showProperties(const IRemoteCommand& remoteCommand, std::string& result);
+
+    CommandReturn getParameter(const IRemoteCommand& remoteCommand, std::string& result);
+
+    CommandReturn setParameter(const IRemoteCommand& remoteCommand, std::string& result);
+
+    CommandReturn getConfigurationParameter(const IRemoteCommand& remoteCommand,
+                                            std::string& result);
+
+    CommandReturn setConfigurationParameter(const IRemoteCommand& remoteCommand,
+                                            std::string& result);
+
+    CommandReturn listBelongingDomains(const IRemoteCommand& remoteCommand, std::string& result);
+
+    CommandReturn listAssociatedDomains(const IRemoteCommand& remoteCommand,
+                                        std::string& result);
+
+    CommandReturn showMapping(const IRemoteCommand& remoteCommand, std::string& result);
+
+    CommandReturn listAssociatedElements(const IRemoteCommand& remoteCommand, std::string& result);
+
+    CommandReturn listConflictingElements(const IRemoteCommand& remoteCommand, std::string& result);
+
+    CommandReturn listRogueElements(const IRemoteCommand& remoteCommand, std::string& result);
+
+    CommandReturn exportConfigurableDomainsToXML(const IRemoteCommand& remoteCommand,
+                                                 std::string& result);
+
+    CommandReturn importConfigurableDomainsFromXML(const IRemoteCommand& remoteCommand,
+                                                   std::string& result);
+
+    CommandReturn exportConfigurableDomainsWithSettingsToXML(const IRemoteCommand& remoteCommand,
+                                                             std::string& result);
+
+    CommandReturn importConfigurableDomainsWithSettingsFromXML(const IRemoteCommand& remoteCommand,
+                                                               std::string& result);
+
+    CommandReturn exportConfigurableDomainWithSettingsFromXML(const IRemoteCommand& remoteCommand,
```

Comment:
oops, this name should be export...**To**XML. (my bad, I did a mistake during the rebase)